x-pack/test/api_integration/apis/actions/builtin_action_types/email.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { KibanaFunctionalTestDefaultProviders } from '../../../../types/providers';
+
+// eslint-disable-next-line import/no-default-export
+export default function emailTest({ getService }: KibanaFunctionalTestDefaultProviders) {
+  const supertest = getService('supertest');
+
+  describe('create email action', () => {
```

Comment:
One thing I noticed which may apply in other tests is there isn't a cleanup function for the data that is created. One way to do this is to add this line between 15 and 16 `after(() => esArchiver.unload('empty_kibana'));`. It will remove any objects created within a namespace that way tests won't pass or fail for the wrong reasons.